examples/pytorch/token-classification/run_ner.py

```diff
@@ -29,17 +29,18 @@
 from datasets import ClassLabel, load_dataset, load_metric
 
 import transformers
+from sparseml_utils import SparseMLTokenClassificationTrainer, TokenClassificationModuleExporter
```

Comment:
Can't we just use the SparseMLTrainer? I see no token classification specific configs